syft/workers/base.py

```diff
@@ -468,7 +468,7 @@ def respond_to_obj_req(self, obj_id: Union[str, int]):
 
         obj = self.get_obj(obj_id)
         if hasattr(obj, "allowed_to_get") and not obj.allowed_to_get():
-            return GetNotPermittedError()
+            raise GetNotPermittedError()
```

Comment:
raising this does not pass the value to the other machine in the case of websocket workers

Reply:
Ok I had missed one line, but now it does: it's all the point of my generic exception serializer